File: sa31/2024-11-06/project/JavaLesson1/src/local/hal/sa31/java/lesson1/ex3/UseNameAndAge.java
package local.hal.sa31.java.lesson1.ex3;

/**
 * SA31 Java文法 Lesson1 Ex3
 *
 * 実行クラス
 *
 * @author Shinzo SAITO
 */
public class UseNameAndAge {
    public static void main(String[] args) {
        NameAndAge nanashi = new NameAndAge();
        nanashi.show();
        NameAndAge taro = new NameAndAge();
        taro.name = "田中太郎";
        taro.age = 28;
        taro.show();
    }
}

File: sa31/2024-11-06/project/JavaLesson1/src/local/hal/sa31/java/lesson1/ex3/useNameAndAge.php
<?php
/**
 * SA31 Java文法 Lesson1 Ex3
 *
 * PHP実行ファイル
 *
 * @author Shinzo SAITO
 */

require_once("NameAndAge.php");

$nanashi = new NameAndAge();
$nanashi->show();
$taro = new NameAndAge();
$taro->name = "田中太郎";
$taro->age = 28;
$taro->show();
